app/forms/applicants/basic_details_form.rb

```diff
@@ -23,9 +23,11 @@ class BasicDetailsForm
     )
 
     validates :national_insurance_number, presence: true
-
     validates :national_insurance_number, format: { with: NINO_REGEXP, message: :not_valid }, allow_blank: true
 
+    validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }, allow_blank: true
+    validates :email, presence: true
+
     # rubocop:disable Lint/DuplicateMethods
```

Comment:
**[This is not part of the PR so don't feel like you need to address it]** If we're just re-defining the reader method for `date_of_birth`, changing the definition of it above from:

```ruby
attr_accessor :date_of_birth
```

to 

```ruby
attr_writer :date_of_birth
```

should get rid of `# rubocop:disable Lint/DuplicateMethods`. 

Unless there's a VERY strong reason to have those, I would avoid using them all together.